model/host/host_test.go

```diff
@@ -1050,3 +1050,368 @@ func TestInactiveHostCountPipeline(t *testing.T) {
+		Status:    evergreen.HostTerminated,
+		Container: "containerId3",
+		ParentId:  "parentId",
+	}
+	host4 := &Host{
+		Id:          "host4",
+		Distro:      distro.Distro{Id: d1},
+		Status:      evergreen.HostRunning,
+		RunningTask: "task2",
+		Container:   "containerId4",
+		ParentId:    "parentId",
+	}
+	host5 := &Host{
+		Id:        "host5",
+		Distro:    distro.Distro{Id: d2},
+		Status:    evergreen.HostProvisioning,
+		Container: "containerId5",
+		ParentId:  "parentId",
+	}
+	host6 := &Host{
+		Id:     "host6",
+		Distro: distro.Distro{Id: d2},
+		Status: evergreen.HostProvisioning,
+	}
+	host7 := &Host{
+		Id:          "host7",
+		Distro:      distro.Distro{Id: d2},
+		Status:      evergreen.HostRunning,
+		RunningTask: "task3",
+	}
+	host8 := &Host{
+		Id:     "host8",
+		Distro: distro.Distro{Id: d2},
+		Status: evergreen.HostRunning,
+	}
+	assert.NoError(host1.Insert())
+	assert.NoError(host2.Insert())
+	assert.NoError(host3.Insert())
+	assert.NoError(host4.Insert())
+	assert.NoError(host5.Insert())
+	assert.NoError(host6.Insert())
+	assert.NoError(host7.Insert())
+	assert.NoError(host8.Insert())
+
+	containers, err := host1.FindAllContainers()
+	assert.NoError(err)
+	assert.Equal(5, len(containers))
+}
+
+func TestFindAllContainersEmpty(t *testing.T) {
+	testutil.HandleTestingErr(db.ClearCollections(Collection), t, "error clearing collections")
+	assert := assert.New(t)
+
+	const d1 = "distro1"
+	const d2 = "distro2"
+
+	host1 := &Host{
+		Id:          "host1",
+		Distro:      distro.Distro{Id: d1},
+		Status:      evergreen.HostRunning,
+		RunningTask: "task",
+	}
+	host2 := &Host{
+		Id:     "host2",
+		Distro: distro.Distro{Id: d1},
+		Status: evergreen.HostStarting,
+	}
+	host3 := &Host{
+		Id:     "host3",
+		Distro: distro.Distro{Id: d1},
+		Status: evergreen.HostTerminated,
+	}
+	host4 := &Host{
+		Id:          "host4",
+		Distro:      distro.Distro{Id: d1},
+		Status:      evergreen.HostRunning,
+		RunningTask: "task2",
+	}
+	host5 := &Host{
+		Id:     "host5",
+		Distro: distro.Distro{Id: d2},
+		Status: evergreen.HostProvisioning,
+	}
+	host6 := &Host{
+		Id:     "host6",
+		Distro: distro.Distro{Id: d2},
+		Status: evergreen.HostProvisioning,
+	}
+	host7 := &Host{
+		Id:          "host7",
+		Distro:      distro.Distro{Id: d2},
+		Status:      evergreen.HostRunning,
+		RunningTask: "task3",
+	}
+	host8 := &Host{
+		Id:     "host8",
+		Distro: distro.Distro{Id: d2},
+		Status: evergreen.HostRunning,
+	}
+	assert.NoError(host1.Insert())
+	assert.NoError(host2.Insert())
+	assert.NoError(host3.Insert())
+	assert.NoError(host4.Insert())
+	assert.NoError(host5.Insert())
+	assert.NoError(host6.Insert())
+	assert.NoError(host7.Insert())
+	assert.NoError(host8.Insert())
+
+	containers, err := host1.FindAllContainers()
+	assert.NoError(err)
+	assert.Equal(0, len(containers))
+}
+
+func TestFindContainers(t *testing.T) {
+	testutil.HandleTestingErr(db.ClearCollections(Collection), t, "error clearing collections")
```

Comment:
right below assert := ..., just do `assert.NoError(db.ClearCollections(Collection))`. This notation is from our older tests. You should also fix this in other places.

Reply:
done